examples/Wealth of Nations.ipynb

```diff
@@ -196,8 +196,8 @@
    },
    "outputs": [],
    "source": [
-    "ax_y = Axis(label='Life Expectancy', scale=y_sc, orientation='vertical', side='left')\n",
-    "ax_x = Axis(label='Income per Capita', scale=x_sc)"
+    "ax_y = Axis(label='Life Expectancy', scale=y_sc, orientation='vertical', side='left', grid_lines='solid')\n",
```

Comment:
Not sure about this, unless you make the grid lines super light. The line won't show, and in this case, I guess you dont care too much about seeing the exact value, more about seeing the relative positions and the paths?